.ci/docker/common/install_executorch.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+set -ex
+
+source "$(dirname "${BASH_SOURCE[0]}")/common_utils.sh"
+
+clone_executorch() {
+  EXECUTORCH_PINNED_COMMIT=$(get_pinned_commit executorch)
+
+  # Clone the Executorch
+  git clone https://github.com/pytorch/executorch.git
+
+  # and fetch the target commit
+  pushd executorch
+  git checkout "${EXECUTORCH_PINNED_COMMIT}"
+  git submodule update --init
+  popd
+
+  chown -R jenkins executorch
+}
+
+install_buck2() {
+  pushd executorch/.ci/docker
+
+  BUCK2_VERSION=$(cat ci_commit_pins/buck2.txt)
+  source common/install_buck.sh
+
+  popd
+}
+
+install_conda_dependencies() {
+  pushd executorch/.ci/docker
+  # Install conda dependencies like flatbuffer
+  conda_install --file conda-env-ci.txt
+  popd
+}
+
+install_pip_dependencies() {
+  pushd executorch/.ci/docker
+
+  # Install all Python dependencies
+  pip_install -r requirements-ci.txt
+  # NB: Just need to install the stable version of these domain libraries here
+  # so that we can install executorch.  They will need to be recompiled later
+  # together with torch
+  pip_install torchvision torchaudio
+
+  popd
+}
+
+setup_executorch() {
+  pushd executorch
+  source .ci/scripts/utils.sh
+
+  install_flatc_from_source
+  pip_install .
+  build_executorch_runner "buck2"
```

Comment:
Would using cmake in release build instead more efficient in CI?

Reply:
Both will work here as cmake is already included.  I remember ET build still uses buck even when cmake is used (to get the source files), so I think just using buck here is easier

Reply:
Yeah, buck is still needed to extract the dep graph. We can only do release build with cmake however.

Reply:
Oh, I see.  I have switched this to cmake